tutorial/src/App.jsx

```diff
@@ -1,40 +1,50 @@
-import { useState } from "react";
+import { useRef, useState } from "react";
 import { useMutation, useQuery } from "../convex/_generated/react";
 
 export default function App() {
   const messages = useQuery("listMessages") || [];
-
-  const [newMessageText, setNewMessageText] = useState("");
   const sendMessage = useMutation("sendMessage");
-
+  const [newMessageText, setNewMessageText] = useState("");
   const [name] = useState(() => "User " + Math.floor(Math.random() * 10000));
+  const messagesList = useRef();
```

Comment:
What is this ref for? Can we remove it?

I love keeping this file short so our tutorial is super simple.

Reply:
Whoops, that's a leftover from having the list scroll when new messages are added. Glad you caught that!